.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: trusty
 language: node_js
 
 node_js:
-  - '6.9.1'
+  - '6.11.0'
```

Comment:
I don't think it matters much within `6` - we definitely haven't decided to move up any majors though